packages/TrilinosInstallTests/find_package_Trilinos/CMakeLists.txt

```diff
@@ -9,3 +9,6 @@ find_package(Trilinos REQUIRED)
 message("Trilinos_FOUND = '${Trilinos_FOUND}'")
 message("Trilinos_SELECTED_PACAKGES_LIST = '${Trilinos_SELECTED_PACAKGES_LIST}'")
 message("Trilinos_LIBRARIES = '${Trilinos_LIBRARIES}'")
+
+add_executable(hello hello.c)
```

Comment:
My only suggestion to make this more portable is to perhaps changes to the a `hello.cpp` C++ file so that the C++ linker will be used.  On some systems, linking with a C linker involving C++ libraries will cause an error.